docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3"
+services:
+  app:
+    build: .
+    ports:
+      - 8080:8080
+    volumes: 
+      - ~/.secrethub:/root/.secrethub
```

Comment:
I'm not a super big fan of this. For me, I get an error because my credential is encrypted with a passphrase. Would be nice to make this configurable or something.

Reply:
Also, for experimenting with stuff I don't like using my personal account credential. Not a super big thing though.